guby_backend/views.py

```diff
@@ -138,23 +138,27 @@ def event_index(request, meetup_id):
 @has_ownership
 def event_add(request, meetup_id):
     form = EventForm(request.POST or None)
-
     if form.is_valid():
-        model = form.save(commit=False)
-        meetup = Meetup.objects.get(id=meetup_id)
-        model.meetup = meetup
-        model.status = Event.STATUS_CREATED
-        #TODO handle errors
-        model.save()
-        return redirect(f'/app/meetups/{meetup_id}/events/')
-
+        try:
```

Comment:
Why does just this handler have a try block?

I can see that you've also removed the `#TODO handle errors` line so I'm assuming it is to handle errors?

Reply:
Yes we need to add a try block all too and handle everything appropriately. Added it for this because I was working on this. @raghothams  will handle it for all the other methods or I can in another PR.

Reply:
Righto. 

The way we should be doing error handling in forms is to inform the user of the errors. There is a way in django to push the errors back to the form. We need to use that approach instead. 

cc @raghothams right?

Reply:
So going by R2's [comment](https://github.com/unnati-xyz/guby/pull/11#issuecomment-664005723), can we revert this try, except block and fix all of them at the same time by passing the errors to the UI?